diff -up libvlcpp-0.1.0/Makefile.am.orig libvlcpp-0.1.0/Makefile.am
--- libvlcpp-0.1.0/Makefile.am.orig	2017-09-04 20:58:39.053566325 +0200
+++ libvlcpp-0.1.0/Makefile.am	2017-09-04 20:58:47.757607764 +0200
@@ -21,7 +21,7 @@ libvlcpp_HEADERS =          \
 	vlcpp/vlc.hpp                 \
 	$(NULL)
 
-pkgconfigdir = $(libdir)/pkgconfig
+pkgconfigdir = $(datadir)/pkgconfig
 pkgconfig_DATA = libvlcpp.pc
 
 if HAVE_EXAMPLES
